zerver/openapi/zulip.yaml

```diff
@@ -11334,6 +11334,23 @@ paths:
                     - 2
                     - 3
                   example: 1
+                web_animate_image_previews:
+                  description: |
+                    Controls how animated images should be played in the message feed.
```

Comment:
"in the web/desktop application" should be in these descriptions.

Reply:
updated!